### PR TITLE
Formating SVM script

### PR DESCRIPTION
**Pull request by TeeSrisantitham**

### CODE REVIEW

File: scripts/svm.sh [hunk 1]
```diff
@@ -8,50 +8,42 @@ svm() {
...
   COMMAND="${1}"
 
-  case $COMMAND in 
-    'create' | 'i')
-      create "${2}"
-      ;;
-    *)
-      hander_err "Command ${COMMAND} is not found"
-      ;;
+  case $COMMAND in
```
Comment (dnfd) [inline scripts/svm.sh:24]: merge these two cases into one

File: scripts/install.sh [hunk 2]
```diff
@@ -30,63 +28,60 @@ svm_detect_profile() {
...
-    echo "=> svm is already installed in $INSTALL_DIR, trying to update the script"
+  if [ -f "${INSTALL_DIR}/sonic.go" ]; then
+    echo "=> svm is already installed in ${INSTALL_DIR}, trying to update the script"
     cd ~/.svm
     git pull origin master
   else
-    echo "=> Cloning svm as script to '$INSTALL_DIR'"
-    git clone --quiet https://github.com/openware/sonic.git  ~/.svm
+    echo "=> Cloning svm as script to '${INSTALL_DIR}'"
+    git clone --quiet https://github.com/openware/sonic.git ~/.svm
```
Comment (dnfd) [inline scripts/install.sh:63]: It should be `git clone --quiet https://github.com/openware/sonic.git $INSTALL_DIR` I think

File: scripts/install.sh [hunk 3]
```diff
@@ -30,63 +28,60 @@ svm_detect_profile() {
...
+svm_clone() {
   local INSTALL_DIR
-  INSTALL_DIR="$HOME/.svm"
+  INSTALL_DIR="${HOME}/.svm"
 
-  if [ -f "$INSTALL_DIR/sonic.go" ]; then
-    echo "=> svm is already installed in $INSTALL_DIR, trying to update the script"
+  if [ -f "${INSTALL_DIR}/sonic.go" ]; then
+    echo "=> svm is already installed in ${INSTALL_DIR}, trying to update the script"
     cd ~/.svm
```
Comment (dnfd) [inline scripts/install.sh:59]: cd $INSTALL_DIR

File: scripts/install.sh [hunk 4]
```diff
@@ -30,63 +28,60 @@ svm_detect_profile() {
...
     printf %s ${HOME}/.svm
   fi
 }
 
-svm_clone ()
-{
+svm_clone() {
   local INSTALL_DIR
-  INSTALL_DIR="$HOME/.svm"
+  INSTALL_DIR="${HOME}/.svm"
```
Comment (dnfd) [inline scripts/install.sh:55]: INSTALL_DIR="$(svm_install_dir)"

File: scripts/svm.sh [hunk 5]
```diff
@@ -8,50 +8,42 @@ svm() {
 
   local i
```
Comment (dnfd) [inline scripts/svm.sh:9]: local i I think noy used anymore

File: scripts/svm.sh (hunk 1 above)
Comment (TeeSrisantitham) [reply scripts/svm.sh]: Ok updated

**Pull request merged**